app/helpers/application_helper.rb

```diff
@@ -10,4 +10,13 @@ def full_title(page_title = '')
     end
   end
 
+  def flash_class(level)
+    case level
```

Comment:
consider using a hash here

Reply:
Let me give that a try! :) Thank you for calling that out.

Reply:
It worked! I'll get that pushed up here shortly. :D